README.md

```diff
@@ -102,6 +106,24 @@ distribution` and `MyLibrary CSS distribution` in the GitLab release.
 `css` files in the `dist` directory and its sub-directories excluding the minified version, plus the
 `build/MyLibrary.zip` file and label it `MyLibrary` in the GitLab release.
 
+#### generics
```

Comment:
I'm wondering whether it might be simpler for users if we would add this functionality to `assets` instead?

```json
assets: [
   { path: 'foo.txt', label: 'Foo', type: 'generic_package' },
   { path: 'bar.txt', label: 'Bar', type: 'project_upload' }
]
```

Reply:
Sounds like a better solution.
Should we define the default value for `type` to `project_upload` to avoid breaking changes or should this be implemented as a breaking change.
Here I would vote against breaking changes here. 😄

Reply:
Yes, I agree that we should not make this a breaking change, thanks for taking care!